src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -29,7 +30,7 @@ public CinemaHallController(CinemaHallService cinemaHallService,
     }
 
     @PostMapping
-    public CinemaHallResponseDto add(@RequestBody CinemaHallRequestDto requestDto) {
+    public CinemaHallResponseDto add(@RequestBody @Valid CinemaHallRequestDto requestDto) {
```

Comment:
What about movie and movie session controllers? we should add DTO validation there too